pkg/gofr/datasource/file/mock_interface.go

```diff
@@ -0,0 +1,191 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: provider_interface.go
+//
+// Generated by this command:
+//
+//	mockgen -source=provider_interface.go -destination=mock_interface.go -package=file
```

Comment:
I can't find any file with name `provide_interface.go`, can you please check from which file is this mock generated.

Reply:
I added the interfaces in that file inside the interface.go file , itself. So that we have one file for interfaces . Updating the mock-gen command.